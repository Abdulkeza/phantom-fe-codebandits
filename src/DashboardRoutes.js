import React from 'react';
import { Route, Routes } from 'react-router-dom';
import Driver from './pages/dashboard/Driver.js';
import Operator from './pages/dashboard/Operator.js';
import RegisterDriver from './pages/forms/RegisterDrivers.js';
import RegisterOperator from './pages/forms/RegisterOperator.js';
import UpdateDriver from './pages/forms/UpdateDriver.js';
import UpdateOperator from './pages/forms/UpdateOperator.js';
import OperatorProfile from './pages/profiles/OperatorProfile.js';

const DashboardRoute = () => {
  return (
    <Routes>
      <Route path="operator" element={<Operator />} />
      <Route path="operator/update" element={<UpdateOperator />} />
      <Route path="driver" element={<Driver />} />
      <Route path="driver/update" element={<UpdateDriver />} />

      <Route path="operator/register" element={<RegisterOperator />} />
      <Route path="driver/register" element={<RegisterDriver />} />
      <Route path="profile" element={<OperatorProfile />} />
    </Routes>
  );
};

export default DashboardRoute;
