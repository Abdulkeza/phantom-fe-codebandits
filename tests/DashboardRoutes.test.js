import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import DashboardRoutes from '../src/DashboardRoutes.js';

describe('DashboardRoutes', () => {
  it('should render the DashboardRoutes component', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <DashboardRoutes />)
        </MemoryRouter>
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
