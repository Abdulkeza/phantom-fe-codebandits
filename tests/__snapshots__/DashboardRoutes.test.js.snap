// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardRoutes should render the DashboardRoutes component 1`] = `")"`;
